README.md

```diff
@@ -87,91 +108,80 @@ echo -e "admin\n${CONCOURSE_ATC_PASSWORD}" | \
    ${FLY_CMD} -t ${DEPLOY_ENV} login -k -c "https://${DEPLOY_ENV}-concourse.cf.paas.alphagov.co.uk"
 ```
```

Comment:
I think we should create a wrapper login script and get rid of this. But this is for another PR.